nixos/modules/module-list.nix

```diff
@@ -1249,6 +1249,15 @@
   ./services/web-apps/node-red.nix
   ./services/web-apps/onlyoffice.nix
   ./services/web-apps/openvscode-server.nix
+  ./services/web-apps/pict-rs.nix
+  ./services/web-apps/peertube.nix
+  ./services/web-apps/plantuml-server.nix
+  ./services/web-apps/plausible.nix
+  ./services/web-apps/pgpkeyserver-lite.nix
+  ./services/web-apps/powerdns-admin.nix
+  ./services/web-apps/prosody-filer.nix
+  ./services/web-apps/matomo.nix
```

Comment:
why are services outside of mobilizon also added ? This looks like a mistake.

Reply:
Oops, looks like the rebase went wrong. I'll fix this